server.R

```diff
@@ -12,218 +22,38 @@ library(purrr)
-              "If your recently updated folder does not appear, please wait for Synapse to sync and refresh"
-            )
-          ),
-          box(
-            status = "primary",
-            solidHeader = TRUE,
-            width = 6,
-            title = "Choose a Metadata Template Type: ",
-            uiOutput("manifest_display_name")
-          )
-        )
-      ),
-      # Third tab item
-      tabItem(tabName = "template",
-        useShinyjs(),
-        h2("Download Template for Selected Folder"),
-        fluidRow(
-          box(
-            title = "Get Link, Annotate, and Download Template as CSV",
-            status = "primary",
-            solidHeader = TRUE,
-            width = 12,
-            actionButton("download", "Click to Generate Google Sheets Template"),
-            hidden(
-              div(
-                id = 'text_div',
-                height = "100%",
-                htmlOutput("text")
-              )
-            ),
-            helpText("This link will leads to an empty template or your previously submitted template with new files if applicable.")
-          ) 
 
-        )
-      ),
-      # Fourth tab content
-      tabItem(tabName = "upload",
-        h2("Submit & Validate a Filled Metadata Template"),
-        fluidRow(
-          box(
-            title = "Upload Filled Metadata as a CSV",
-            solidHeader = TRUE,
-            status = "primary",
-            width = 12,
-            uiOutput('fileInput_ui')
-          ),
-          box(
-            title = "Metadata Preview",
-            solidHeader = TRUE,
-            status = "primary",
-            width = 12,
-            DT::DTOutput("tbl")
-            ),
-          box(
-            title = "Validate Filled Metadata",
-            status = "primary",
-            solidHeader = TRUE,
-            width = 12,
-            actionButton("validate", "Validate Metadata"),
-            hidden(
-              div(id = 'text_div2', 
-                  height = "100%",
-                  htmlOutput("text2")
-              ),
-              DT::DTOutput("tbl2"),
-              actionButton("gsheet_btn", "  Click to Generate Google Sheet Link", icon = icon("table")),
-              div(id = 'gsheet_div', 
-                  height = "100%",
-                  htmlOutput("gsheet_link")
-              )
-            ),
-            helpText(
-              HTML("If you have an error, please try editing locally or on google sheet.<br/>
-                   Reupload your CSV and press the validate button as needed.")
-            )
-          ),
-          box(title = "Submit Validated Metadata to Synapse",
-              status = "primary",
-              solidHeader = TRUE,
-              width = 12,
-              uiOutput("submit")
-          )
-        )
-      )
-    ),
-    uiOutput("Next_Previous"),
+# Don't necessarily have to set `RETICULATE_PYTHON` env variable
+reticulate::use_condaenv("data_curator_env")
```

Comment:
does not match with env name in README: `data_curator_env_oauth`

Reply:
Good catch, thanks. I've changed the environment name to `data_curator_env`.